test/integration_test.py

```diff
@@ -484,6 +487,12 @@ def run(thread_count):
         # Reset to pre-test state.
         portal_service.overwrite(old_db)
 
+    def _test_check_all_indices(self):
```

Comment:
This looks confusing. _test_indexing wouldn't pass if indices were missing. The reason I asked for this test was not so much to assert that all indices exist (even though that is useful) but to assert that ESClientFactory.get() works. I am ok keeping this test and even the name under two conditions: it is invoked before _test_indexing and a comment is added to emphasize the dual purpose of this test. Before asking me for another review, have someone help you with the comment, by asking for a peer review.